### ISSUE TITLE
Bug with connecting 2 and more components in Modelling Editor

### ISSUE THREAD

**Issue by HardMax71** [ls1intum/Artemis#1556]
### Describe the bug
After adding two or more components/interfaces on the already existing component, creating any visible connection between added elements is not possible (Arrows between components do not show up).
After dragging out one element from the boundaries of the bigger element, a part of connection is shown up.

#### To Reproduce
1.Go to T02E02 or T02E03 Excercise (EIST) in Artemis
2. Click on "open modelling editor"
3. Add new component (let's say C)
4. Increase the size of C to the extent when it can have more than 1 interfaces/components in it.
5. Add 2 new components (let's say, A and B) to the previously created component.
6. Connect A and B (type of connection isn't important).
7. No new visible connections between A and B are visible.
8. Drag A out of the boundaries of C
9. Now the part of connection between A and B is visible.

#### Expected behavior
After step 6 it's expected to have new visible connection (black line) between A and B.

#### Screenshots

After step 4
![image](https://user-images.githubusercontent.com/66359507/83619732-f86f2d80-a594-11ea-82f3-d0e21c5d481d.png)
After step 5
![image](https://user-images.githubusercontent.com/66359507/83619817-1177de80-a595-11ea-8826-5d460ba5dc5e.png)
After step 6
![image](https://user-images.githubusercontent.com/66359507/83619867-218fbe00-a595-11ea-86d8-a38fa8ed3b88.png)
After step 7
![image](https://user-images.githubusercontent.com/66359507/83619940-41bf7d00-a595-11ea-8182-5f12baf0105b.png)
After step 9
![image](https://user-images.githubusercontent.com/66359507/83620144-95ca6180-a595-11ea-8404-cd2e7b9aa49b.png)

### Environment

<details><pre>
 - OS: Windows 10
 - Browser: Google Chrome 
 - Version: 83.0.4103.61 (Official Build) (64-bit)
</pre></details>

#### Additional context

This bug also comes in H04E04, but there all the connections between components are "hidden" (on the layer lower).
That's how it looks like :
![image](https://user-images.githubusercontent.com/66359507/83620759-7253e680-a596-11ea-8297-655d88d363c7.png)

**Comment by jpbernius** [comment]
I tested this on https://apollon.ase.in.tum.de/ and cannot reproduce this in the standalone version. @TobiasPr Can you have a look at this?

**Comment by tessaRuckstuhl** [comment]
The same happens with functional models - the associations are drawn below the system border. Should I create a separate issue for that, too? @TobiasPr

**Comment by TobiasPr** [comment]
@tessaRuckstuhl its the same issue
